<?xml version="1.0"?>
<launch>

    <!-- Load parameters -->
    <rosparam command="load" ns="" file="$(find table_plane_extractor)/config/config.yaml"/>
    <node name="table_plane_extractor_server" pkg="table_plane_extractor" type="table_plane_extractor_srv.py"  output="screen"/>
</launch>